app/models.py

```diff
@@ -26,6 +27,9 @@ def insert_roles():
             'User': (
                 Permission.GENERAL, 'main', True
             ),
+            'AgencyWorker': (
+                Permission.WORKER, 'main', True
```

Comment:
Do we not want to give `AgencyWorker` it's own blueprint? Just like `Administrator` has it's `admin` blueprint?